library/Zend/XmlRpc/Client.php

```diff
@@ -43,7 +43,7 @@ class Zend_XmlRpc_Client
      * HTTP Client to use for requests
      * @var Zend_Http_Client
      */
-    protected $_httpClient = null;
+    protected $_httpClient;
```

Comment:
It's never set to null (if passed as null in the constructor, gets set to a new `Zend_Http_Client` object). Cleaner to remove null here.